packages/backend/api/getSegmentColors/getSegmentColors.go

```diff
@@ -30,9 +30,9 @@ import (
 // @Success 304 {none} nil "No changes: The data has not been modified since the last request date provided in the 'If-Modified-Since' header."
 // @Failure 500 "Internal Server Error: Error during the processing of the request."
 //
-// @Router /risk-prediction/getSegmentColors [get]
+// @Router /risk-prediction/segmentColors [get]
 func GetSegmentColors(c echo.Context) error {
-	logger.Log.Info().Msg("GET /risk-prediction/getSegmentColors")
+	logger.Log.Info().Msg("GET /risk-prediction/segmentColors")
```

Comment:
should we align our url casing while we're at it? segment-colors?

Reply:
yeah good point, i will fix this